// Try catch

const numero1 = 20;
const numero3 = 20;

// try va a intentar ejecutar el codigo y si hay un error lo imprime pero no termina la ejecucion
// es util para no detener la ejecucion total de un codigo, ya que solo muetra donde esta el error
// util para promises
try {
    console.log(numero2);
} catch(error) {
        console.log(error)
}
